include/signal.h

```diff
@@ -43,7 +43,7 @@
 
 #define NULL_SIGNAL_SET ((sigset_t)0x00000000)
 #define ALL_SIGNAL_SET  ((sigset_t)0xffffffff)
-#define MIN_SIGNO       0
+#define MIN_SIGNO       1
 #define MAX_SIGNO       31
```

Comment:
The title says  SIGRTMIN to SIGUSR1 

But isn't this the bit position? Not the mapping?

Reply:
> 
> 
> The title says SIGRTMIN to SIGUSR1
> 
> But isn't this the bit position? Not the mapping?

I misdescribed it, i will modify this title. sorry~